src/BulletSoftBody/btDeformableBodySolver.cpp

```diff
@@ -401,7 +402,7 @@ void btDeformableBodySolver::predictMotion(btScalar solverdt)
 			}
 		}
 	}
-	m_objective->applyExplicitForce(m_residual);
+	applyExplicitForce();
```

Comment:
are the changes to the original btDeformableBodySolver needed?

Reply:
This came from some debugging code, and it is no longer needed